src/components/swap/SwapButtons/SwapButton.tsx

```diff
@@ -52,9 +56,27 @@ const LogoWithText = styled.div`
   justify-content: center;
 `
 
-const StyledPlataformText = styled(Text)`
-  text-transform: none;
+const UnifyFontSize = css`
   font-size: 15px;
+
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+    font-size: 13px;
+  `};
+`
+
+const StyledPlatformText = styled.div`
+  text-transform: none;
```

Comment:
I think we should use uppercase to avoid:

<img width="527" alt="image" src="https://user-images.githubusercontent.com/1926216/183665974-2cec3309-f7a8-4849-ad71-b239b42436ce.png">

Reply:
It was like that before, so up to you guys

Reply:
I prefer upper case.